airbyte-webapp/src/components/connection/CatalogTree/next/CatalogTreeTableHeader.tsx

```diff
@@ -80,9 +80,6 @@ export const CatalogTreeTableHeader: React.FC = () => {
       <HeaderCell size="small">
         <FormattedMessage id="sources.sync" />
       </HeaderCell>
-      {/* <TextCell>
```

Comment:
Suppose this is fine to remove for now?  We do want to have this field once we introduce column selection to the new table (https://github.com/airbytehq/airbyte/pull/21058) we want this column.  Unless the designs have changed and I'm not up to date.

Reply:
I just removed it since it had been sitting around. I can re-add 👍